README.md

```diff
@@ -2,26 +2,21 @@
 
 This is a standalone exercise to practice writing tests and engaging in Test Driven Development (TDD). Clone this repository and then open the resulting `DevOps-Course-Bank-API` folder in VSCode. Open it as the project folder - don't open a parent folder and then browse down to the repository folder, otherwise subsequent instructions here will not work.
 
-Read through this file to get the project running and then follow the instructions in [during_the_workshop.md](./during_the_workshop.md) for the exercise instructions.
+Read through this file to get the project running and then follow the instructions in [exercise_instructions.md](./exercise_instructions.md).
 
 ## Setup
 
-This project requires Python 3.8 (or higher) and uses Poetry for Python to handle dependencies. Follow the instructions 
-[here](https://python-poetry.org/docs/#installation) to set up Poetry.
+This project requires Python 3.8 (or higher) and uses pip to handle dependencies. If you are familiar with using virtual environments to manage dependencies, use your usual workflow to create and activate a virtual environment before proceeding.
```

Comment:
Looks like I missed this line when updating the minimum to 3.9 recently ahead of deprecation
```suggestion
This project requires Python 3.9 (or higher) and uses pip to handle dependencies. If you are familiar with using virtual environments to manage dependencies, use your usual workflow to create and activate a virtual environment before proceeding.
```